style/desktop.css

```diff
@@ -240,12 +241,330 @@
+
+  .about-section {
+    background-color: #1c1a19;
+    justify-content: start;
+    align-items: center;
+    position: relative;
+    margin-top: 5rem;
+    padding: 5rem 8rem 0.6rem 8rem;
+  }
+
+  .bor-line {
+    border-bottom: 1px solid #fff;
+    width: 90%;
+    height: 100%;
+  }
+
+  .about-container {
+    height: 386px;
+    width: 566px;
+    margin-left: 4rem;
+    margin-top: 5rem;
+    align-items: start;
+    gap: 1rem;
+  }
+
+  .about-head {
+    width: 566px;
+    height: 78px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 88px;
+    color: #ff6b00;
+  }
+
+  .about-detail {
+    width: 500px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+    margin-bottom: 2rem;
+  }
+
+  .resume-btn {
+    padding: 0.8rem 1.5rem;
+  }
+
+  .about-illustration {
+    height: 600px;
+    width: 1000px;
+    background-image: url("/src/assets/imgs/illustration-about-me@2x-desktop.png");
+    background-repeat: no-repeat;
+    background-size: 90%;
+    background-position: center;
+    position: absolute;
+    top: 50%;
+    right: 50%;
+    transform: translate3d(100%, -50%, 0);
+  }
+
+  .lan-frameworks {
+    background-color: #1c1a19;
+    text-align: center;
     justify-content: center;
     align-items: start;
-    background-color: orangered;
+    padding: 6rem 0 6rem 0;
+    gap: 8rem;
+  }
+
+  .lan-frameworks h3 {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    margin-top: 1rem;
+    margin-bottom: 2rem;
+    color: #fff;
+  }
+
+  .lan-skill {
+    width: 20rem;
+    height: 20rem;
+    margin-top: 1rem;
+    gap: 0.5rem;
+  }
+
+  .lan-skill li {
+    text-align: left;
+    list-style: none;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    margin-bottom: 1rem;
+    color: #fff;
+  }
+
+  .lan-icon img {
+    width: 16px;
+    height: 16px;
+    margin-bottom: 1rem;
+  }
+
+  .lan-icon {
+    width: 20rem;
+    height: 3rem;
+    display: flex;
+    justify-content: start;
+    align-items: center;
+    gap: 1rem;
+  }
+
+  .get-touch {
+    grid-template-columns: repeat(2, 1fr);
+    justify-content: center;
+    align-items: center;
+    padding: 0 10rem;
+    position: relative;
+  }
+
+  .get-text {
+    max-width: 455px;
+    width: 100%;
+    height: 40rem;
+    padding-top: 8rem;
+    text-align: left;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+  }
+
+  .contact-form {
+    max-width: 65rem;
+    width: 100%;
+    height: 40rem;
+    padding-top: 8rem;
+    justify-content: start;
+    align-items: center;
+    gap: 0.5rem;
+  }
```

Comment:
As mentioned above, I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

>### The Figma design
>
><img width="648" alt="Screenshot 2023-05-04 at 5 29 30 AM" src="https://user-images.githubusercontent.com/91237525/236166555-55019795-4ddd-462e-8f23-4fd7ab145935.png">
>
>### Your design
>
><img width="1436" alt="Screenshot 2023-05-04 at 5 29 44 AM" src="https://user-images.githubusercontent.com/91237525/236166600-5e631c16-e036-40ab-80fb-66e8e089edb5.png">

Kindly make sure the mobile version is the same as it is in the Figma design

>### This is how your mobile version looks like after designing the Desktop version
>
><img width="372" alt="Screenshot 2023-05-04 at 5 37 26 AM" src="https://user-images.githubusercontent.com/91237525/236168243-042857fc-f853-4967-b055-ac865a7e0240.png">
><img width="370" alt="Screenshot 2023-05-04 at 5 32 54 AM" src="https://user-images.githubusercontent.com/91237525/236168059-211733ee-c218-41bc-b63d-e5f5fc11b2d7.png">
>
>### The Figma design
>
><img width="388" alt="Screenshot 2023-05-04 at 5 38 41 AM" src="https://user-images.githubusercontent.com/91237525/236168515-13b9b13f-6133-4f63-a93d-37b4503ef1a7.png">
><img width="389" alt="Screenshot 2023-05-04 at 5 40 34 AM" src="https://user-images.githubusercontent.com/91237525/236168874-8f1e7a99-820e-4c5d-9ef2-294996f42c1e.png">

Reply:
## what i did 
* fixed the contact form for mobile version
* also the headline illustration
> 
![headline](https://user-images.githubusercontent.com/99332080/236203170-4a15b5a0-9d6a-443a-aeef-61df37b49276.png)
![contact-from](https://user-images.githubusercontent.com/99332080/236203214-1a0afec8-9164-46c0-9032-e16ca695b4ca.png)